### PR TITLE
Add Ionoscloud support

### PR DESCRIPTION
**Pull request by adeatcu-ionos**

### CODE REVIEW

**Comment by adeatcu-ionos** [comment]
@sergeylanzman can you please take a look? thank you!

File: terraformutils/utils.go [hunk 1]
```diff
@@ -96,6 +96,8 @@ func RefreshResources(resources []*Resource, provider *providerwrapper.ProviderW
 
 	wg.Wait()
 	for _, r := range resources {
+		log.Printf("Instance state %v  \n", r.InstanceState)
```
Comment (sergeylanzman) [inline terraformutils/utils.go:99]: please delete, no need here

File: terraformutils/utils.go [hunk 2]
```diff
@@ -106,6 +108,8 @@ func RefreshResources(resources []*Resource, provider *providerwrapper.ProviderW
 	for _, resourceGroup := range slowProcessingResources {
 		for i := range resourceGroup {
 			r := resourceGroup[i]
+			log.Printf("slow processing Instance state %v  \n", r.InstanceState)
```
Comment (sergeylanzman) [inline terraformutils/utils.go:111]: please delete, no need here

**Review by sergeylanzman — CHANGES_REQUESTED** [review]
please delete unused prints

File: terraformutils/utils.go (hunk 1 above)
Comment (adeatcu-ionos) [reply terraformutils/utils.go]: @sergeylanzman done in c07585a114788b9a428a7d601d501196d4306e73

File: terraformutils/utils.go (hunk 2 above)
Comment (adeatcu-ionos) [reply terraformutils/utils.go]: @sergeylanzman done in c07585a114788b9a428a7d601d501196d4306e73

**Review by MariaSu — APPROVED** [review]

**Review by MariaSu — APPROVED** [review]

**Review by sergeylanzman — APPROVED** [review]

**Comment by adeatcu-ionos** [comment]
@MariaSu @sergeylanzman can you run the `golangci-lint` again, please? I looked into the previous outputs and I fixed all the errors that were related to our changes.

**Pull request merged**